dinghy-lib/src/apple/xcode.rs

```diff
@@ -24,52 +29,29 @@ pub fn add_plist_to_app(bundle: &BuildBundle, arch: &str, app_bundle_id: &str) -
         "<key>CFBundleIdentifier</key><string>{}</string>",
         app_bundle_id
     )?;
-    writeln!(plist, "<key>UIRequiredDeviceCapabilities</key>")?;
-    writeln!(plist, "<array><string>{}</string></array>", arch)?;
     writeln!(plist, "<key>CFBundleVersion</key>")?;
     writeln!(plist, "<string>{}</string>", arch)?;
     writeln!(plist, "<key>CFBundleShortVersionString</key>")?;
     writeln!(plist, "<string>{}</string>", arch)?;
-    writeln!(plist, "<key>UILaunchStoryboardName</key>")?;
-    writeln!(plist, "<string></string>")?;
-    writeln!(plist, r#"</dict></plist>"#)?;
-    /*
-    let app_name = app_bundle_id.split(".").last().unwrap();
-    let app_path = app_path.as_ref().join(format!("{}.app", app_name));
-    println!("WRAP AS APP: {:?} -> {:?}", executable.as_ref(), app_path);
-    let _ = fs::remove_dir_all(&app_path);
-    fs::create_dir_all(&app_path)?;
-    fs::copy(&executable, app_path.join(app_name))?;
-
-
-    let mut plist = fs::File::create(app_path.join("Info.plist"))?;
-    writeln!(plist, r#"<?xml version="1.0" encoding="UTF-8"?>"#)?;
-    writeln!(plist, r#"<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">"#)?;
-    writeln!(plist, r#"<plist version="1.0"><dict>"#)?;
-    writeln!(
-    plist,
-    "<key>CFBundleExecutable</key><string>{}</string>",
-    app_name
-    )?;
-    writeln!(
-    plist,
-    "<key>CFBundleIdentifier</key><string>{}</string>",
-    app_bundle_id
-    )?;
-    writeln!(plist, "<key>UIRequiredDeviceCapabilities</key>")?;
-    writeln!(
-    plist,
-    "<array><string>{}</string></array>",
-    target.split("-").next().unwrap()
-    )?;
+    match sim_type {
+        // The iOS simulator and the iOS device have the same plist.
+        Some(AppleSimulatorType::Ios) | Some(AppleSimulatorType::Tvos) | None => {
+            writeln!(plist, "<key>UIRequiredDeviceCapabilities</key>")?;
+            writeln!(plist, "<array><string>{}</string></array>", arch)?;
+            writeln!(plist, "<key>UILaunchStoryboardName</key>")?;
+            writeln!(plist, "<string></string>")?;
+        },
```

Comment:
In the future, these will diverge more I think. Today, I spent a bit of time diving into [an audio microphone issue on cpal](https://github.com/RustAudio/cpal/issues/842#issuecomment-1957652903). I manually added `micrphone` to the [`UIRequiredDeviceCapabilities`](https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/iPhoneOSKeys.html#//apple_ref/doc/uid/TP40009252-SW3).

It's out of the scope of this PR but it'd be nice to have some possible permission options or maybe a custom `Info.plist`.